orbstation/code/objects/vending/clothing.dm

```diff
@@ -73,6 +73,6 @@
 				/obj/item/lipstick/jade = 2,
 				/obj/item/lipstick/green = 2,
 				/obj/item/lipstick/black = 2,
-            ),
+        	),
```

Comment:
was this indentation stuff intentional or is this just git being weird

Reply:
git bein weird. no idea why it does that.